.eslintrc.js

```diff
@@ -44,7 +44,13 @@ module.exports = {
     "jsx-a11y/label-has-for": "warn", // see #173
     "no-console": "warn",
     "no-extra-semi": "off", // because it isn't that important
-    "react/prop-types": "warn" // see #175
+    "react/prop-types": "warn", // see #175
+    "jsx-a11y/no-onchange": "warn",
```

Comment:
how many warnings are there?  Can you fix it?  These are accessibility things, and the goal is to avoid adding stuff here if we can.

Reply:
see below